cmd/registry/auth/oauth/oauth.go

```diff
@@ -61,7 +60,12 @@ func NewOauthCmd(ctx context.Context, opt *options.CommonOptions) *cobra.Command
 		SilenceErrors:         true,
 		SilenceUsage:          true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return o.RunOauth(ctx, args)
+			reg := args[0]
```

Comment:
I would move this logic to `func (o *RegistryOauthOptions) RunOauth(ctx context.Context, args []string)`, and keep that function. Just to be consistent with the other commands.

Reply:
will move it into its on function 👍